apps/dot_ext/forms.py

```diff
@@ -22,17 +21,6 @@ def __init__(self, user, *args, **kwargs):
         agree_label = u'Yes I have read and agree to the <a target="_blank" href="%s">API Terms of Service Agreement</a>*' % (
             settings.TOS_URI)
         super(CustomRegisterApplicationForm, self).__init__(*args, **kwargs)
-        choices = []
```

Comment:
@aviars @ekivemark @samgensburg-gov would a preferable option here be to set the form field to hidden and default all scopes to selected? If we do that I think we should add some validation guarenteeing that a created application has at least one `ProtectedCapability` (scope) as the token auth flow fails silently if no scopes are associated with the application.